buildtest/cli/report.py

```diff
@@ -954,3 +958,17 @@ def print_report_summary_output(
     console.print(table)
     pass_results.print_report(title="PASS Tests", color=color)
     fail_results.print_report(title="FAIL Tests", color=color)
+
+
+def report_detailed(report, configuration):
+    """This method will print detailed summary test results which can be retrieved via ``buildtest report --detailed`` command
+    Args:
+        report (buildtest.cli.report.Report): An instance of Report class
+        configuration (buildtest.config.SiteConfiguration): Instance of SiteConfiguration class that is loaded buildtest configuration.
+    """
+    detailed_results = Report(
+        format_args="name,id,user,state,returncode,runtime,outfile,errfile,buildspec",
+        report_file=report.reportfile(),
+        configuration=configuration,
+    )
+    detailed_results.print_report(count=1)
```

Comment:
please remove `count=1` this is not correct. We don't want `buildtest report --detailed` to show only one entry. It should show all all the rows.